src/components/builder/validate/i18n.tsx

```diff
@@ -9,12 +9,12 @@ import {BuilderContext} from '@/context';
 import {DataMap, Panel, Tab, TabList, TabPanel, Tabs, TextField} from '../../formio';
 
 export function useManageValidatorsTranslations<S extends SchemaWithValidation>(
-  keys: PossibleValidatorErrorKeys<S>[]
+  keys: PossibleValidatorErrorKeys<S>[],
+  field: string = 'translatedErrors'
```

Comment:
so, to avoid weir things, I don't think this should be typed as `string` and even the field as a whole should be specified - probably it's better to allow a prefix (which can be anything):

```tsx
// function signature
keys: PossibelValidatorErrorKeys<S>[],
prefix: string = '',

// function body
const fieldName = `${prefix}${prefix ? '.' : ''}translatedErrors`;
```

The rest of the type information looks like it can remain as is, however [L64](https://github.com/open-formulieren/formio-builder/pull/167/files#diff-e1c3cd11dc4c171ab1f0efa43e03b841daaaab0141ce38725838bd6272ec462dR64) will need to include the prefix too I think, since the form field name must be the deeply nested sub component schema.